share/scripts/inverse/run_gromacs.sh

```diff
@@ -133,14 +153,15 @@ else
   echo "${0##*/}: No walltime defined, so no time limitation given to $mdrun"
 fi
 
-#>gmx-5.1 has new handling of bonded tables
+#gmx-5.1 has new handling of bonded tables
+
 if [[ ${mdrun_opts} != *tableb* ]]; then
   tables=
   for i in table_[abd][0-9]*.xvg; do
     [[ -f $i ]] && tables+=" $i"
   done
   if [[ -n ${tables} ]]; then
-	  msg --color blue --to-stderr "Automatically added '-tableb${tables} to mdrun options (add -tableb option to cg.inverse.gromacs.mdrun.opts yourself if this is wrong)"
+    msg --color blue --to-stderr "${0##*/}: Automatically added '-tableb${tables} to mdrun options (add -tableb option to cg.inverse.gromacs.mdrun.opts yourself if this is wrong)"
```

Comment:
The is true for the logfile, but not for `msg`, which go the screen.  None of the other 91 `msg` has that.